eventloop_unix.go

```diff
@@ -125,6 +125,10 @@ func (el *eventloop) loopOpen(c *conn) error {
 }
 
 func (el *eventloop) loopRead(c *conn) error {
+	if !c.opened {
```

Comment:
I don't think this is necessary, if the conn is closed, there will be an error when calling the following unix.Read, which will be handled properly.

Reply:
I provide the test to confirm that error can be raised here. You can see this condition covered in report.
Also I have errors from my own servers with stacktraces where NPE caused in this method.

Reply:
NPE will be not in loopRead but in Decode or React if you will try to call for example 'ResetBuffer' or 'Read' on a closed conn.

Reply:
Huh, I recheck it now. And I will dig deeper because I have a NPE but currently it not exposed in test :)
After some time I'll provide more comments and remove unnecessary conditions if I cannot reproduce NPEs.

Reply:
Currently I cannot reproduce it. So I think it would be better to remove that condition and create new issue if I can reproduce NPE in readLoop.